custom/icds_reports/ucr/tests/test_beneficiary_form.py

```diff
@@ -1,16 +1,18 @@
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
-
 from mock import patch
+
 from custom.icds_reports.ucr.tests.test_base_form_ucr import BaseFormsTest
 
 
+@patch('custom.icds_reports.ucr.expressions._get_user_location_id',
+       lambda user_id: 'qwe56poiuytr4xcvbnmkjfghwerffdaa')
+@patch('corehq.apps.locations.ucr_expressions._get_location_type_name',
+       lambda loc_id, context: 'supervisor')
```

Comment:
I attempted to pull these patches out to some common code, but I was getting into an issue where the tests were no longer being picked up by the auto discover. Decided to punt for now